packages/astro/client.d.ts

```diff
@@ -172,6 +172,10 @@ declare module 'astro:components' {
 	export * from 'astro/components';
 }
 
+declare module 'astro:schema' {
```

Comment:
I see the addition here but I don't see the types removal for z from astro:actions. You only removed their implementation I think or I missed something

Reply:
ah yes, all good then!